http3/server.go

```diff
@@ -265,6 +294,23 @@ func (s *Server) handleRequest(sess quic.Session, str quic.Stream, decoder *qpac
 		s.logger.Infof("%s %s%s", req.Method, req.Host, req.RequestURI)
 	}
 
+	// Adjust the read deadline if necessary
+	if setReadDeadline && !hdrDeadline.Equal(wholeReadDeadline) {
```

Comment:
I believe we can remove this, since H2 doesn't use read deadlines. What do you think?

Reply:
Yes, sure.